packages/core-generators/src/renderers/typescript/templates/types.ts

```
@@ -7,12 +7,100 @@ import {
 import { TsCodeFragment } from '../fragments/types.js';
 import { z } from 'zod';
 import { CASE_VALIDATORS } from '@baseplate-dev/utils';
+import { templateConfigSchema } from '@baseplate-dev/sync/extractor-v2';
```

ESLint

[error] Expected "@baseplate-dev/sync/extractor-v2" to come before "@baseplate-dev/utils".

(perfectionist/sort-imports)